### PR TITLE
Version - 1

### PR DESCRIPTION
**Pull request by heintayzar-hm**
- The user logs in to the website, only by typing the username (a proper authenticated login).
- In the navigation panel, the user can see links to:
    - Planes.
    - "My reservations".
    - "Add the plane that you selected as a theme".
    - "Delete the plane that you selected as a theme".
- On the main page, the user can see [planes](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005577).
  
- When the user selects a specific item, they can see the [details page](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005579) with its full description.
    - On the details page, the user can click the "Reserve" button (in the design you can see the "Configure" button - please replace it with the "Reserve" button).

- When the user clicks the "Add Item" link in the navigation panel they can see a form for adding a new item.

- Make the app responsive, creating both mobile and desktop versions.

### Note As we are 5 members teams, there are some things that we changed,

- On the Navbar there are no delete planes link anymore and it will be on the respective details page.

### Pull Request for React

https://github.com/V-Blaze/gvash-planes/pull/17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rspec spec --format documentation
+```
```
Comment (Meltrust) [inline README.md:170]: - [ ] When I run the tests according to instructions on your readme, I get these 2 fails:

image

If these are bugs, kindly fix them so your tests pass.  If these are not bugs, kindly update your readme instructions with specific steps to run your tests properly.

That way, all your tests will pass for all your users, and your app's integrity will be secured.  💪

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :users
+  post 'auth/login', to: 'authentication#login' , as: 'login'
+  namespace :api do
+    namespace :v1 do
+      resources :planes
```
Comment (Meltrust) [inline config/routes.rb]: - [ ] OPTIONAL: Nice routes and nice use of the `#resources` method. However, you are generating more routes than needed.

## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources):



## Example

For the planes controller, I can see that you are only using index, create, show, and destroy (no edit or update).

So:

```ruby
resources :planes, only: [:index, :create, :show, :destroy]

```

You already did these steps nicely with the planes_reservations route.

That way, your routes will be more consistent, and only the ones needed will be generated💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @heintayzar-hm,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Almost all tests are passing✔️
-  Nice code organization ✔️ 
-  API is working well✔️ 
-  Superb readme ✔️ 
-  Nice model validations ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Review by AHP15 — APPROVED** [review]
Hi Team, 

Our project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

**Pull request merged**